internal/orchestrator/ray/cluster/ssh.go

```diff
@@ -334,8 +334,8 @@ func generateRayClusterConfig(cluster *v1.Cluster, imageRegistry *v1.ImageRegist
 	}
 
 	initializationCommands := []string{}
-	// set registry CA.
-	if imageRegistry.Spec.Ca != "" {
+	// only set registry CA if user is root.
+	if rayClusterConfig.Auth.SSHUser == "root" && imageRegistry.Spec.Ca != "" {
```

Comment:
what will happen if SSH user is not root?

Reply:
no permission, we need to create a directory in /etc/docker/ and write the certificate file, which requires higher permissions